binds/python/bind_immutable.cpp

```diff
@@ -70,8 +70,7 @@ static void bind_immutable_module(py::module &m) {
             "Note: perimeters belonging to the n'th section are located at indices:\n"
             "[Morphology.sectionOffsets(n), Morphology.sectionOffsets(n+1)[")
         .def_property_readonly("section_offsets", [](const morphio::Morphology& morpho){
-                return py::array(static_cast<py::ssize_t>(morpho.sectionOffsets().size()),
-                                 morpho.sectionOffsets().data());
+                return as_pyarray(morpho.sectionOffsets());
```

Comment:
sectionOffsets is declared:
`const std::vector<uint32_t> sectionOffsets() const;` (which is itself weird), but I'm surprised that `as_pyarray` deals with the `const` part without complaint - do we not have enough warnings on?

Reply:
This is an incredible remark! Thank you very much @mgeplf 

The `const` return type of member function `sectionOffsets` is not a problem for `as_pyarray` as the constness it  part of the `sequence` **template** parameter type.

The real question here is "what is going on in the function body?"
```
Sequence* seq_ptr = new Sequence(std::move(seq));
```
In our case `Sequence` is `const std::vector<uint32>`. And because `std::move(seq)` is
a const rvalue reference, it is not possible to use the move constructor (that expects a non-const rvalue reference), and it falls back to ... the copy constructor!

Here is a snippet that highlights the case: https://godbolt.org/z/2qR9jb

I will remove the const return type of `sectionOffsets` in a minute.
Thanks again

Reply:
> I will remove the const return type of sectionOffsets in a minute

Good call, I had the same thing in my version; I was using the version of libsonata's asArray, but it doesn't have the `static_cast<py::ssize_t>` in it, and I was trying to decide how annoyed I was by that...

All that to say you beat me to the PR, and I appreciate you doing it :)